Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -181,19 +209,27 @@ function createFocusableComponent(Component: any) {
       if (this.props.componentRef) {
         this.props.componentRef(null);
       }
-      this._isMounted = false;      
+      this._isMounted = false;
     }
 
     _splitProps(props: Object) {
       this._focusableProps = {};
       this._componentProps = {};
-  
+
       for (const key in props) {
         if (key in FocusableWindowsTemplate.focusablePropTypes) {
           // Property supported by WindowsControl
           this._focusableProps[key] = props[key];
+
+          // Accessibility properties are exposed to the WindowsControl but also must be set on the Component.
+          if (key === 'importantForAccessibility' ||
```

Comment:
@reseul This is intentional. What happens here is that from whoever uses the Component, the FocusableWindows remains completely out-of-control. For accessibility, the user will work and set the properties on the Component. The FocusableWindow just mimics these and there are basically two cases that both work fine - 
1. When the IFA hides children - like Yes/NoHideDesc - where the FocusableWindow will hide the Component (and in that case  Component will have no effect)
2. When IFA doesn't hide children - No/Auto that results in logic falling through that is also OK. 
It is important that the accessibility properties are set on Component, too, because we want for it to appear in the Accessibility tree the way it was intended to appear by dev (in terms of properties) even if it doesn't participate in accesibility, i.e. it may be set as "Raw" but when the tree is inspected we want to see a "button" and not "view", so traits must be set on it.
Setting event on both is OK. It will be fired only if the element is "current" for narrator and that depends on how the dev sets the flag in the application code.

---
In reply to: [173046472](https://github.com/Microsoft/react-native-windows/pull/1698#discussion_r173046472) [](ancestors = 173046472)